astropy/io/ascii/core.py

```diff
@@ -363,7 +363,22 @@ def join(self, vals):
 
 
 def _replace_tab_with_space(line, escapechar, quotechar):
-    """Replace tab with space within ``line`` while respecting quoted substrings"""
+    """Replace tabs with spaces in given string, preserving quoted substrings
+    
+    Parameters
+    ----------
+    line : str
+        String containing tabs to be replaced with spaces.
+    escapechar : str
+        Character in `line` used to escape special characters.
+    quotechar: str
+        Character in `line` indicating the start/end of a substring.
+        
+    Returns
+    -------
+    ''.join(newline) : str
```

Comment:
Saying `''.join(newline)` is perhaps a little too much detail.  Just `line` would be fine.